packages/std-client/tsconfig.json

```diff
@@ -14,5 +14,5 @@
     "skipLibCheck": true,
     "jsx": "react-jsx"
   },
-  "include": ["src"]
+  "include": ["src", "../store-sync/src/recs/common.test-d.ts"]
```

Comment:
for context, why is this needed?

Reply:
oops, this got added by vscode after a rename